### PR TITLE
runtime: fix example code in README.md (fixes #691)

### LINKED ISSUE #691: compilation error in wasmer-runtime example
Trying to run the example code in `wasmer-runtime` fails with the following complication error:

```
error[E0034]: multiple applicable items in scope
  --> amethyst_scripting/src/lib.rs:26:10
   |
26 |         .call(&[Value::I32(42)])?;
   |          ^^^^ multiple `call` found
   |
   = note: candidate #1 is defined in an impl for the type `wasmer_runtime_core::typed_func::Func<'_, (_,), _>`
   = note: candidate #2 is defined in an impl for the type `wasmer_runtime_core::typed_func::Func<'_, (), _>`
   = note: candidate #3 is defined in an impl for the type `wasmer_runtime_core::typed_func::Func<'_, _, _>`
   = note: candidate #4 is defined in an impl for the type `wasmer_runtime_core::typed_func::Func<'_, (_, _), _>`
   = note: and 10 others
```

### PR DESCRIPTION
**Pull request by ryzokuken**

### CODE REVIEW

File: lib/runtime/README.md [hunk 1]
```diff
@@ -83,8 +83,7 @@ fn main() -> error::Result<()> {
     let mut instance = instantiate(WASM, &import_object)?;
 
     let values = instance
-        .func("add_one")?
```
Comment (Hywan) [inline lib/runtime/README.md]: It's OK to just replace `func` by `dyn_func`. That's what I would recommend. Can you update the PR accordingly please?

Comment (ryzokuken) [reply lib/runtime/README.md]: I'm sorry I was a little occupied in the last few weeks. Updating the PR ASAP.

Comment (syrusakbary) [reply lib/runtime/README.md]: @ryzokuken awesome! Let us know once is ready to re-review :)

**Comment by syrusakbary** [comment]
Closing the PR in favor of #855 855

**Comment by ryzokuken** [comment]
Thanks! Sorry I got a little occupied to finish this up nicely.

**Pull request closed without merging**